### PR TITLE
Simple Loading collection

### PR DESCRIPTION
**Pull request by Pratichhya**

### CODE REVIEW

**Comment by jdries** [comment]
- there's again the basics notebook in this PR
- maybe add cloud masking? That's also part of any Sentinel-2 workflow and make the result look nicer if you do max time composite.
- We could also load the resulting tiff and visualize it in the notebook, as an extra?

File: openeo/Load_Collection.ipynb [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    "\n",
+    "# connect to the backend\n",
+    "connection = openeo.connect(url = \"openeo-staging.creo.vito.be\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To verify whether they have been authenticated to load collections, users can check their connection to the backend. If the connection is established, they can be confident that they are authenticated and can proceed with loading the collections."
```
Comment (soxofaan) [inline openeo/Load_Collection.ipynb:29]: > If the connection is established, they can be confident that they are authenticated and can proceed with loading the collections.

This is misleading: a connection can be "established" also without authentication

File: openeo/basics.ipynb [hunk 2]
```diff
@@ -12,132 +12,522 @@
...
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "print(connection.list_collections()[1])"
+    "connection.list_collections()[1]"
```
Comment (soxofaan) [inline openeo/basics.ipynb:149]: I don't think using `connection.list_collections()[1]`  is something to suggest in a "basics" notebook, it's not very practical and the dict dump is very ugly/unreadable.

Comment (Pratichhya) [reply openeo/basics.ipynb]: updated

File: openeo/Load_Collection.ipynb (hunk 1 above)
Comment (Pratichhya) [reply openeo/Load_Collection.ipynb]: updated

**Comment by Pratichhya** [comment]
> * there's again the basics notebook in this PR
> * maybe add cloud masking? That's also part of any Sentinel-2 workflow and make the result look nicer if you do max time composite.
> * We could also load the resulting tiff and visualize it in the notebook, as an extra?

The basic notebook comes from the main branch. Removal of this notebook results in github action issue. 
The cloud masking steps are included in the notebook. 
Visualization of the result is added and also sample output are uploaded to the repo.

**Comment by soxofaan** [comment]
>> there's again the basics notebook in this PR
>
> The basic notebook comes from the main branch. 

The problem is not that there is a basic notebook. 
I think the point of Jeroen is that it's a bad idea to touch this same notebook in each PR ( #1 #2 #3 #4 #5 ). You'll get too easily into merge conflict troubles that way (especially with jupyter notebooks)

> Removal of this notebook results in github action issue.

I'm not sure what you mean with "github action issue".

**Comment by Pratichhya** [comment]
@soxofaan . Thank you for your suggestions. I have the collection file with the suggested formatter (nevertheless using 4 indentation for load_collection).

Also I get your and Jeroen's point. I will try to fix the changes with basic notebook in other pull request

**Pull request merged**